packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -270,6 +284,17 @@ export class Wallet {
   async importPrivKey (privkey: string, label: string = '', rescan: boolean = true): Promise<void> {
     return await this.client.call('importprivkey', [privkey, label, rescan], 'number')
   }
+
+  /**
+   * Get detailed information about in-wallet transaction
+   *
+   * @param {string} txid transaction id
+   * @param {boolean} includeWatchOnly optional, default = true
+   * @return {Promise<InWalletTransaction>}
+   */
+  async getTransaction (txid: string, includeWatchOnly: boolean = true): Promise<InWalletTransaction> {
+    return await this.client.call('gettransaction', [txid, includeWatchOnly], 'number')
```

Comment:
in this case we need use `bignumber` to prevent precision loss on floating point because the response contains `amount`

Reply:
Made the change to this, to apply `bignumber` on `amount`.